pages/account/manage/event/[[...data]].js

```diff
@@ -72,6 +65,14 @@ export default function ManageEvent({ BASE_URL, event }) {
   );
   const [price, setPrice] = useState(event.price?.startingFrom || 0);
 
+  const formatDate = (inputDate) => {
+    const d = new Date(inputDate);
+    const date = d.toISOString().split("T")[0];
+    const time = d.toLocaleTimeString();
+
+    return `${date}T${time}`;
+  };
```

Comment:
What is the reason for moving this code?

Reply:
I wanted to make the code clearer, by keeping all the useStates together. Other than that, there was no particular reasons to do it. I can leave it as it was before the change. Let me know :)

Reply:
Yes please, it is best to keep it focused and we can get it merged

Reply:
Done 👍 
Sorry for the inconvenience, and thank you for your constant feedback, I'm learning a lot thanks to this project !

Reply:
Thank you for your consistent efforts to push this forwards! Let's merge it 👍